.github/workflows/test.yml

```diff
@@ -26,6 +26,14 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python }}
+      - name: Build Substrait planloader library
+        run: |
+          cd ${{ github.workspace }}/third_party/substrait-cpp
+          make release
+      - name: Install Substrait planloader library
+        run: |
+          cd ${{ github.workspace }}/third_party/substrait-cpp/build-Release/export/planloader
+          make install
```

Comment:
We'll want to handle compilation via the `pyproject.toml` file, I'll leave some notes there.